packages/kotti-ui/source/kotti-field-number/KtFieldNumber.vue

```diff
@@ -94,7 +94,7 @@ export default defineComponent({
 					maximum: null,
 					minimum: props.minimum,
 					offset: -props.step,
-					value: props.value,
+					value: field.currentValue,
```

Comment:
given that this is the most important part of this PR, i'd heavily recommend changing the title to be fix KtFieldNumber `isDecrementEnabled` b/c that's the thing we should definitely know has been fixed above all, right?

Reply:
Agreed